### DISCUSSION TITLE
help

### DISCUSSION THREAD

**Discussion by rimaelfala** [BuildingComponentLibrary/bcl-manifest#28] [Q&A]
hello everyone , i couldnt download mesures for my open studio 3.2.0 ,any one can help me with that please?

**Comment by Smit199782** [comment]
I guess the server is down, so not able to download either.

**Comment by kflemin** [comment]
BCL is online now...let me know if you are still unable to download measures. thanks!

**Reply by Smit199782** [reply]
Yes it worked out.
